### DISCUSSION TITLE
ImageMagick:

### DISCUSSION THREAD

**Discussion by dcutaia2** [ImageMagick/ImageMagick#5161] [Digital Image Processing]
![Screenshot 2022-05-17 162418](https://user-images.githubusercontent.com/105815865/169147090-5fa1ceaf-c65c-432e-b945-25e66628206c.png)

Here is the ImageMagick command I am using:
Imagick convert C:\ImageMagick\Etchings\XXXXXXX_XXXXXXX_building_bitmap_line_1t_dx_0x_bks_x_x_x_x_x_x_x_x_x_ba_5100.pdf[0] -quiet -units PixelsPerInch -density 150 -filter LanczosSharp -distort Resize 1024x1024^ +antialias -extent 1024x1024   C:\ImageMagick\Etchings\20220517\XXXXXXX_XXXXXXX_poweruser_png_copyright_jostens_lanczosSharp512.png

The image on the left is the original pdf file and the image on the right is the png that ImageMagick created.  The ImageMagick png file that is created has its lines broken apart and not lines.  Does anyone have a recommendation on how to use ImageMagick to keep or retain the lines that are in parallel? 

Thanks,
DAC

**Comment by snibgo** [comment]
> The image on the left is the original pdf file ...

No, it isn't. It is a PNG file. Perhaps it was made from a PDF file.

I suggest you show the actual PDF file.

Perhaps a larger "-density" would help.

**Reply by fmw42** [reply]
Perhaps extract the raster from the PDF directly using `pdfimages`.
